packages/salesforcedx-vscode-core/test/vscode-integration/util/authInfo.test.ts

```diff
@@ -5,26 +5,26 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import {
-  AuthInfo,
-  ConfigAggregator,
-  Connection,
-  StateAggregator
-} from '@salesforce/core';
-import { OrgConfigProperties } from '@salesforce/core/lib/exported';
```

Comment:
since moving the configAggProvider to utils we can't directly stub that method in core.